src/generate_fastapi/mako_templates/src/project/domain/model/template_schemas.py

```diff
@@ -26,6 +26,7 @@ class _Base(pydantic.BaseModel):
 % for field in model.fields:
     % if not field.is_primary_key:
     % if field.field_type.nullable:
+    < %! from generate_fastapi.translators.json_translator import JsonTranslator % >
```

Comment:
According to the specs there shouldn't be a space between `<` or `>` and `%`

Reply:
The import turned out to already be present in the template. I also fixed a new line error most likely caused by the linter